docs/install.md

```diff
@@ -0,0 +1,189 @@
+
+To check if you've done this properly, open a new `cmd` or `powershell` instance and enter `file`. You should see output similar to the following:
+
+```
+C:\Users\USERNAME>file
+Usage: file [-bcCdEhikLlNnprsSvzZ0] [--apple] [--extension] [--mime-encoding]
+            [--mime-type] [-e <testname>] [-F <separator>]  [-f <namefile>]
+            [-m <magicfiles>] [-P <parameter=value>] [--exclude-quiet]
+            <file> ...
+       file -C [-m <magicfiles>]
+       file [--help]
+
+```
+
+This is currently the only method we recommend. **We do NOT recommend `scoop install file`**, since Scoop's `file` cannot handle Unicode file names properly (e.g. `pexels-oliver-sjöström-1433052.jpg`).
+
+\*\* See https://www.c-sharpcorner.com/article/add-a-directory-to-path-environment-variable-in-windows-10/ for help on adding a directory to the Windows path.
+
+\*\* Note that adding this directory to the Windows path will make other Linux programs (such as `cat`, `ls`, and `vim`) that come from `/usr/bin/` accessible to `cmd` and `powershell`.
+
+### Installation
+
+First, install Scoop: https://scoop.sh/. Then, in a new `powershell` instance, enter:
+
+```sh
+scoop install yazi unar jq poppler fd ripgrep fzf zoxide
+```
+
+Yazi on Scoop is maintained by community contributors and may not always have the latest version available. You may want to download the latest binary from [Yazi's GitHub Releases](https://github.com/sxyazi/yazi/releases) if that is the case.
+
+### Image Previews
+
+Currently, only WezTerm and Mintty (i.e., Git Bash, which comes with Git for Windows) support images, and [Yazi has adapted them](https://github.com/sxyazi/yazi#image-preview) to work right out of the box! Windows Terminal does not yet support images though, see https://github.com/microsoft/terminal/issues/5746.
+
+## AOSC OS
+
+```bash
+sudo oma install yazi
+```
+
+## Official binaries
+
+You can download the latest official binaries from [GitHub Releases](https://github.com/sxyazi/yazi/releases).
+
+## Cargo
+
+Setup the latest stable Rust toolchain:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Now you can install `yazi-fm` from [crates.io](https://crates.io/crates/yazi-fm):
+
+```bash
+cargo install --locked yazi-fm
+```
+
+Or install the latest git version:
+
+```bash
+cargo install --locked --git https://github.com/sxyazi/yazi.git
+```
+
+If it fails to build, please check if `make` and `gcc` is installed on your system.
+
+## Build from source
+
+Setup the latest stable Rust toolchain:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Clone the repository and build Yazi:
+
+```bash
+git clone https://github.com/sxyazi/yazi.git
+cd yazi
+cargo build --release
+```
+
+Then, you can run:
+
+```bash
+./target/release/yazi
+```
+
+If it fails to build, please check if `make` and `gcc` is installed on your system.
```

Comment:
I think each of the headings should be for the platform/app you use to install it, not what OS it is on. I.e. Homebrew, Pacman, Cargo, Scoop, etc instead of macOS, Arch, Windows.